examples/hooks/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useBreadcrumb, UseBreadcrumbProps } from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type BreadcrumbProps = React.ComponentProps<'div'> & UseBreadcrumbProps;
+
+export function Breadcrumb(props: BreadcrumbProps) {
+  const { items, refine, createURL } = useBreadcrumb(props);
+
+  return (
+    <div
+      className={cx(
+        'ais-Breadcrumb',
+        items.length === 0 && 'ais-Breadcrumb--noRefinement',
```

Comment:
We can rely on [`canRefine`](https://github.com/algolia/instantsearch.js/blob/8b0af3e44f3cbb891623cab5cfb32e4cf9046f93/src/connectors/breadcrumb/connectBreadcrumb.ts#L218) for this check. (Same applies in other places.)